project-resources/PROJECT_FUNDS.md

```diff
@@ -0,0 +1,59 @@
+# OpenJS Foundation Project-specific Initiative Funds
+
+The OpenJS Foundation (OpenJSF) provides a set of monetary funds for projects of the OpenJSF. 
+These funds are designed to support collaboration and incentive project-specific initiatives that have high-impact within the individual project's scope.
```

Comment:
> Templates work well for issues, but they're clumsy for PRs. Even more so, a PR needs to effect some change.

I wonder if requests should be opened as an issue then? The idea behind a PR is so that either:

- A very visible way of voting against something
- The file change would maybe be a document we use for "logging/auditing" past approved funds, similar to how we do with expense requests for travel funds.

I'm open for suggestions of course :)

Reply:
PR is better than issue, because it allows for tracking approval much better and leaves an explicit trace in the repo when merged.

As I mention in my previous comment, my suggestion would be to imitate our pre-existing process on travel funds.

Reply:
Hmm, so we are on the same page, you'd like me to add to this PR a Markdown file with a table similar to the one from the Travel Expenses one?

Just trying to understand what are the next steps here :)

Reply:
Yes. We already have one process for asking for funds, and it works. Let's make sure that this process matches that one, rather than creating a different process.

The instructions for requesting travel funds have stood the test of time rather well, let's effectively reuse them here.

Reply:
Hey, @eemeli, I've just pushed the changes! I apologise for the delay.